lib/main_containers.pm

```diff
@@ -106,6 +106,11 @@ sub load_compose_tests {
     loadtest('containers/compose', run_args => $run_args, name => $run_args->{runtime} . "_compose");
 }
 
+sub load_seccomp_tests {
```

Comment:
I don't see the point of this subroutine. seccomp is one single test run and it's not expected to ever grow.

I would at this point not create a own subroutine for this but just use `loadtest` everywhere. If at some point int time this grows to include more testmodules, we can then make a subroutine.

Reply:
Fixed.